site-src/guides/infrastructure.md

```diff
@@ -0,0 +1,45 @@
+# Gateway infrastructure labels and annotations
+
+??? success "Standard Channel since v1.2.0"
```

Comment:
Is this right?

Reply:
For the record, that syntax has always looked weird to me, too, but it does render correctly in the Netlify preview. 🙂